2024-11-17 14:53:53,884: Error running WSGI application
2024-11-17 14:53:53,887: ModuleNotFoundError: No module named 'zoneinfo'
2024-11-17 14:53:53,888:   File "/var/www/brandocs_pythonanywhere_com_wsgi.py", line 44, in <module>
2024-11-17 14:53:53,888:     from app import app as application
2024-11-17 14:53:53,889: 
2024-11-17 14:53:53,889:   File "/home/Brandocs/brandocs/./app.py", line 12, in <module>
2024-11-17 14:53:53,889:     from zoneinfo import ZoneInfo  # Using zoneinfo instead of pytz
2024-11-17 14:53:53,889: ***************************************************
2024-11-17 14:53:53,890: If you're seeing an import error and don't know why,
2024-11-17 14:53:53,890: we have a dedicated help page to help you debug: 
2024-11-17 14:53:53,890: https://help.pythonanywhere.com/pages/DebuggingImportError/
2024-11-17 14:53:53,890: ***************************************************
2024-11-17 14:53:56,260: Error running WSGI application
2024-11-17 14:53:56,261: ModuleNotFoundError: No module named 'zoneinfo'
2024-11-17 14:53:56,261:   File "/var/www/brandocs_pythonanywhere_com_wsgi.py", line 44, in <module>
2024-11-17 14:53:56,261:     from app import app as application
2024-11-17 14:53:56,261: 
2024-11-17 14:53:56,262:   File "/home/Brandocs/brandocs/./app.py", line 12, in <module>
2024-11-17 14:53:56,262:     from zoneinfo import ZoneInfo  # Using zoneinfo instead of pytz
2024-11-17 14:53:56,262: ***************************************************
2024-11-17 14:53:56,262: If you're seeing an import error and don't know why,
2024-11-17 14:53:56,263: we have a dedicated help page to help you debug: 
2024-11-17 14:53:56,263: https://help.pythonanywhere.com/pages/DebuggingImportError/
2024-11-17 14:53:56,263: ***************************************************
2024-11-17 14:53:56,972: Error running WSGI application
2024-11-17 14:53:56,973: ModuleNotFoundError: No module named 'zoneinfo'
2024-11-17 14:53:56,973:   File "/var/www/brandocs_pythonanywhere_com_wsgi.py", line 44, in <module>
2024-11-17 14:53:56,974:     from app import app as application
2024-11-17 14:53:56,974: 
2024-11-17 14:53:56,974:   File "/home/Brandocs/brandocs/./app.py", line 12, in <module>
2024-11-17 14:53:56,974:     from zoneinfo import ZoneInfo  # Using zoneinfo instead of pytz
2024-11-17 14:53:56,975: ***************************************************
2024-11-17 14:53:56,975: If you're seeing an import error and don't know why,
2024-11-17 14:53:56,975: we have a dedicated help page to help you debug: 
2024-11-17 14:53:56,975: https://help.pythonanywhere.com/pages/DebuggingImportError/
2024-11-17 14:53:56,975: ***************************************************
2024-11-17 14:53:58,124: Error running WSGI application
2024-11-17 14:53:58,125: ModuleNotFoundError: No module named 'zoneinfo'
2024-11-17 14:53:58,125:   File "/var/www/brandocs_pythonanywhere_com_wsgi.py", line 44, in <module>
2024-11-17 14:53:58,125:     from app import app as application
2024-11-17 14:53:58,125: 
2024-11-17 14:53:58,126:   File "/home/Brandocs/brandocs/./app.py", line 12, in <module>
2024-11-17 14:53:58,126:     from zoneinfo import ZoneInfo  # Using zoneinfo instead of pytz
2024-11-17 14:53:58,126: ***************************************************
2024-11-17 14:53:58,126: If you're seeing an import error and don't know why,
2024-11-17 14:53:58,127: we have a dedicated help page to help you debug: 
2024-11-17 14:53:58,127: https://help.pythonanywhere.com/pages/DebuggingImportError/
2024-11-17 14:53:58,127: ***************************************************
2024-11-17 14:54:06,028: Error running WSGI application
2024-11-17 14:54:06,029: ModuleNotFoundError: No module named 'zoneinfo'
2024-11-17 14:54:06,029:   File "/var/www/brandocs_pythonanywhere_com_wsgi.py", line 44, in <module>
2024-11-17 14:54:06,029:     from app import app as application
2024-11-17 14:54:06,030: 
2024-11-17 14:54:06,030:   File "/home/Brandocs/brandocs/./app.py", line 12, in <module>
2024-11-17 14:54:06,030:     from zoneinfo import ZoneInfo  # Using zoneinfo instead of pytz
2024-11-17 14:54:06,030: ***************************************************
2024-11-17 14:54:06,030: If you're seeing an import error and don't know why,
2024-11-17 14:54:06,031: we have a dedicated help page to help you debug: 
2024-11-17 14:54:06,031: https://help.pythonanywhere.com/pages/DebuggingImportError/
2024-11-17 14:54:06,031: ***************************************************